cli/pkg/printer/resources.go

```diff
@@ -540,3 +543,106 @@ type billingIssue struct {
 	Metadata     string `header:"metadata" json:"metadata"`
 	EventTime    string `header:"event_time,timestamp(ms|utc|human)" json:"event_time"`
 }
+
+func (p *Printer) PrintResource(resources map[string]map[string][]*runtimev1.Resource) {
```

Comment:
In any other case, we most likely do not want to include org and project names when listing resources (e.g. `rill project status`). It's confusing to keep it here as a generic function, someone else may try to reach for it when they need to list resources. It would be better to just keep the printing logic in `dump_resources.go`.